tasks/__init__.py

```diff
@@ -623,6 +623,13 @@ def clear_sessions(ctx, months=1, dry_run=False):
     clear_sessions.clear_sessions_relative(months=months, dry_run=dry_run)
 
 
+@task
+def end_prereg_challenge(ctx, dry_run=False):
```

Comment:
Why was this added as an invoke task? Doesn't seem necessary when the script can be tested by calling it directly.